gradle/lib-admin.gradle

```diff
@@ -20,19 +20,42 @@ def applyExcludedTasks()
     gradle.includedBuild( 'lib-admin-ui' ).getLoadedSettings().getStartParameter().setExcludedTaskNames( excludedTasks )
 }
 
+def applyBuildTaskDependencies()
+{
+    def libAdminBuildTask = gradle.includedBuild( 'lib-admin-ui' ).task( ':build' )
+
+    if ( hasTask( 'copyDevResources' ) ) {
+        copyDevResources.dependsOn libAdminBuildTask
+    }
+    if ( hasTask( 'typescript' ) ) {
+        typescript.dependsOn libAdminBuildTask
+        typescript.mustRunAfter(libAdminBuildTask)
```

Comment:
I don't see the need for `mustRunAfter`, if task is already `dependsOn`
At least doc does not tell anything about mustRunAfter https://docs.gradle.org/current/userguide/composite_builds.html#included_build_task_dependencies

Reply:
@rymsha Yeah, I told @alansemenov about it and asked him not to approve it yet. I forgot to remove the `mustRunAfter` (was testing both), but was AFK at the moment I noticed it, so I couldn't update the PR.